packages/flutter_tools/templates/app/android.tmpl/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.2-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.1.1-all.zip
```

Comment:
This is a breaking version bump. Is there anything in our wider build that's going to be broken by the changes in 5? I can see that CI is failing on this PR with some Gradle issue, looks related to me at first glance.

https://docs.gradle.org/5.0/userguide/upgrading_version_4.html suggests running `gradle help --scan` to see what's being used in our build that's currently deprecated, might help debug CI here.

Reply:
There's a single deprecation warning coming from AndroidPlugin in Gradle 6.0: https://scans.gradle.com/s/sn4bo7b2wijbg

Reply:
Ran a scan, and there's nothing reported that we need to change, but I can reproduce the failing test locally.  It's due to Gradle stdout and stderr being empty when the app/build.gradle file contains errors.  The gradlew process correctly exits with non-zero status, but there's no output as to why - and turning on verbose warnings or info doesn't help ☹️

Reply:
:( Just a guess, but my gut instinct would be that something subtle has changed in Gradle's interior logic that probably impacts how it handles breakages in our projects' build files now. Might be impacted by us doing work outside of the normal execution order in `afterEvaluate`.

Reply:
Can we take a pass at just updating the android plugin and defer updating the gradle version?

Reply:
I can confirm that commenting out our plugin import and the `flutter { }` block prints the error message expected in the test. 
`Could not find method builTypes() for arguments`, and `BUILD FAILED`.   

Also, last week I started experimenting with a new Gradle script after reading your doc @mklim.

Reply:
@dnfield only to version 3.3.2 -- https://developer.android.com/studio/releases/gradle-plugin#updating-gradle